db/migrate/20220722101202_add_reference_to_planning_applications.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class AddReferenceToPlanningApplications < ActiveRecord::Migration[6.1]
+  def up
+    add_column :planning_applications, :reference, :string
+    add_index :planning_applications, :reference
+
+    PlanningApplication.all.find_each do |planning_application|
+      planning_application.send(:set_reference)
```

Comment:
Not sure this can find a private method.
My migration failed locally for this

Reply:
That's weird...I thought using `send` had fixed that 🤔